lighthouse-core/config/default-config.js

```diff
@@ -516,6 +517,7 @@ const defaultConfig = {
         {id: 'is-on-https', weight: 2, group: 'pwa-installable'},
         {id: 'service-worker', weight: 1, group: 'pwa-installable'},
         {id: 'installable-manifest', weight: 2, group: 'pwa-installable'},
+        {id: 'ios-pwa-icon', weight: 1, group: 'pwa-installable'},
```

Comment:
as above, I don't believe this is required for installability (though we should check). Would be `pwa-optimized` group in that case.

We also have talked about making the current group "Chrome installable" and having separate groups for Firefox, Safari, etc as we add them so we don't end up with a mixed browser checklist, which can be kind of confusing when testing. We should revisit that decision before (if) adding it to the current installable list